lib/capybara_discoball/runner.rb

```diff
@@ -20,7 +20,21 @@ def boot
 
       private
 
-      def with_webrick_runner
+      def with_webrick_runner(&block)
+        retry_count = 3
+
+        begin
+          launch_webrick(&block)
+        rescue Errno::EADDRINUSE => e
+          raise if retry_count <= 0
+
+          retry_count -= 1
+          puts e.inspect
+          retry
```

Comment:
What do you think about rearranging this block to utilize an `if ... else` block?